integration_tests/pages/page.ts

```diff
@@ -4,23 +4,19 @@ import logAccessibilityViolations from '../support/logAccessibilityViolations'
 export type PageElement = Cypress.Chainable<JQuery>
 
 export default abstract class Page {
-  static verifyOnPage<T>(constructor: new () => T): T {
-    return new constructor()
+  static verifyOnPage<T, Options>(constructor: new (options: Options) => T, options?: Options): T {
+    return new constructor(options)
   }
 
-  static verifyOnPageTitle = <T>(constructor: new (string) => T, title?: string): T => {
```

Comment:
Removing this variant of `verifyOnPage` because the original can handle any custom parameters.